<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginpage</class>
 <widget class="QDialog" name="loginpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Futura</family>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>loginpage</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#loginpage{
background-image:url(&quot;/Users/shashinoorghimire/Qt/projects/travelSphere/images/background&quot;);
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>457</x>
     <y>70</y>
     <width>791</width>
     <height>580</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
background:#B3F4D4;
border-radius:50px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
font-family:Futura;
color:balck;
font-size:15px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
color:black;
font-family:Helvetica;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>491</x>
     <y>119</y>
     <width>614</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color:black;
}</string>
   </property>
   <property name="text">
    <string>Username/Email-address</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>491</x>
     <y>250</y>
     <width>614</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color:black;
}</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>472</x>
     <y>162</y>
     <width>761</width>
     <height>54</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
background-color:white;
border-radius:22px;
color:black;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>472</x>
     <y>290</y>
     <width>761</width>
     <height>54</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
background-color:white;
border-radius:22px;
color:black;
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>738</x>
     <y>403</y>
     <width>229</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
border-radius:22px;
background-color:#5B7279;
color:white;
}</string>
   </property>
   <property name="text">
    <string>LOGIN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>469</y>
     <width>366</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura</family>
     <pointsize>20</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
border-radius:12px;
background-color:#5B7279;
color:white;
}</string>
   </property>
   <property name="text">
    <string>Don't have an account? Sign Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>738</x>
     <y>519</y>
     <width>229</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
border-radius:13px;
background-color:#5B7279;
color:white;
}</string>
   </property>
   <property name="text">
    <string>Sign in with Google</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>252</y>
     <width>251</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>85</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color:#5B7279;
}</string>
   </property>
   <property name="text">
    <string>Pick &amp;
Go</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>331</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
background-image:url(&quot;/Users/shashinoorghimire/Qt/projects/travelSphere/images/logo-login.png&quot;);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
